lib/scoreboard.js

```diff
@@ -1,8 +1,44 @@
 module.exports = ScoreBoard
 
-function ScoreBoard (name) {
-  this.name = name
-  this.displayText = null
-  this.items = {}
-  this.position = null
+const sortItems = (a, b) => {
+  if (a.value > b.value) return -1
+  if (a.value < b.value) return 1
+  return 1
+}
+
+function ScoreBoard (packet) {
```

Comment:
It would be more coherent with the rest of the code to use the "class" keyword for this here
Optional change though, what you did works too

Reply:
I'll change that if we refactor everything to classes as I proposed in #801